gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,324 @@
+
+Replace `<path-to-registration-file>` with the path and filename of the existing `registration.yaml` file. Typically, the existing registration file is overwritten by the renewal process. You can optionally output the new registration file to a directory by using the `--output-directory` flag.
+
+For Flex Gateway running in a container, invoke `renew` as part of the container `run` command:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl \
+-v "$(pwd)":/renew \
+-v <path-to-registration-directory>:/registration \
+-u $UID mulesoft/flex-gateway \
+renew \
+--client-id=<your-client-id> \
+--client-secret=<your-client-secret> \
+--output-directory=/renew \
+/registration/<registration-filename>
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl --userns=keep-id \
+-v "$(pwd)":/renew:Z \
+-v <path-to-registration-directory>:/registration:Z \
+-u $UID docker.io/mulesoft/flex-gateway \
+renew \
+--client-id=<your-client-id> \
+--client-secret=<your-client-secret> \
+--output-directory=/renew \
+/registration/<registration-filename>
+----
+====
+
+Invoking the command results in the following output:
+
+[source,text]
+----
+Starting registration renewal, please be patient.
+Registration renewal completed, the configuration files were written in directory ".".
+For security, modify the file permissions to restrict production scenario access to the user running flex.
+
+Side effect: the following files were written in the current directory
+registration.yaml
+----
+
+[[inspection-cli-overview]]
+== Registration Inspection CLI Command Overview
+
+Flex Gateway includes the following CLI command to inspect registration:
+
+----
+flexctl registration inspect
+----
+
+The command inspects your Flex Gateway registration, extracting useful information you use to verify whether the renewal command was successful.
+
+Invoke the following `help` command for information about usage:
+
+[source,ssh]
+----
+flexctl registration inspect --help
+----
+
+For Flex Gateway running in a container, invoke `help` as part of the container `run` command:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl mulesoft/flex-gateway inspect --help
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl docker.io/mulesoft/flex-gateway inspect --help
+----
+====
+
+Refer to the following topics for more information about inspection command usage:
+
+* <<inspect-with-custom-url,Inspect Registration Status with a Custom URL>>
+* <<inspect-with-reg-file,Inspect Registration Status with a Registration File>>
+
+[[inspect-with-custom-url]]
+== Inspect Registration Status with a Custom URL
+
+You can inspect a registration by specifying the base URL of the Flex Gateway Admin API. The default value is `http://localhost:9999`. Omit the `--flex-gateway-admin-api-url` flag if using the default value.
```

Comment:
I would also clarify that if no flags are passed, by default it will look up on the flex gateway admin api